src/PTY.ts

```diff
@@ -65,9 +81,11 @@ export function executeCommand(
             ...execOptions,
             env: _.extend({PWD: directory}, process.env),
             cwd: directory,
-            shell: "/bin/bash",
+            shell: "cmd",
```

Comment:
Honestly I'm not sure what's this option impact; probably need to use some portable path definition from `loginShell`.